.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env", "@babel/preset-react"]
```

Comment:
Here we could use the presets we had talked about, but tried following `@babel/**` pattern to match your guide